ringpop.go

```diff
@@ -226,12 +258,23 @@ type BootstrapOptions struct {
 
 // Bootstrap starts the Ringpop
 func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
+
```

Comment:
I consider this a great victory for mankind :|

On Mon, Dec 7, 2015 at 10:29 AM, Daniel Simmons <notifications@github.com>
wrote:

> In ringpop.go
> <https://github.com/uber/ringpop-go/pull/34#discussion_r46800196>:
>
> > @@ -226,12 +258,23 @@ type BootstrapOptions struct {
> >
> >  // Bootstrap starts the Ringpop
> >  func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
> > +
>
> Yep, already did it:
> 587528f
> <https://github.com/uber/ringpop-go/commit/587528f5811bb555c549abb0c6ea23513a7d472b>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uber/ringpop-go/pull/34/files#r46800196>.
>